kitsune/sumo/jinja2/base.html

```diff
@@ -252,11 +252,13 @@ <h5 class="mzp-c-footer-heading">{{ _('Firefox for Developers') }}</h5>
         </section>
 
         <section class="mzp-c-footer-col">
-          <h5 class="mzp-c-footer-heading">{{ _('Firefox Accounts') }}</h5>
```

Comment:
I believe this is already part of the main branch 
https://github.com/mozilla/kitsune/blob/main/kitsune/sumo/jinja2/base.html#L255